app/controlplane/internal/conf/controlplane/config/v1/conf.proto

```diff
@@ -168,19 +168,6 @@ message CA {
   }
 }
 
-// OnboardingSpec is a configuration to automatically onboard users in organizations with specific roles
-message OnboardingSpec {
```

Comment:
onboarding spec is still very specific from controlplane app, what is the minimum common denomitator that needs to be used somewhere else?